api/config/v1alpha1/helpers.go

```diff
@@ -248,3 +258,13 @@ func (r *EnvoyGatewayProvider) GetEnvoyGatewayKubeProvider() *EnvoyGatewayKubern
 
 	return r.Kubernetes
 }
+
+func DefaultEnvoyGatewayAdmin() *EnvoyGatewayAdmin {
+	return &EnvoyGatewayAdmin{
+		Debug: false,
+		Address: &EnvoyGatewayAdminAddress{
+			Port: 8899,
```

Comment:
IMO should use constant instead of magic

Reply:
Done